tests/AssetManagerTest/Resolver/AggregateResolverTest.php

```diff
@@ -8,6 +8,11 @@
 
 class AggregateResolverTest extends PHPUnit_Framework_TestCase
 {
+    public function setUp()
+    {
+        require_once __DIR__ . '/../../_files/ResolverWithCollect.php';
```

Comment:
Actually, this class is not needed: just `$this->getMock('...ResolverInterface', ['collect'])`